### ISSUE TITLE
uninstall playbook failed - unable to delete docker

### ISSUE THREAD

**Issue by yuvalif** [openshift/openshift-ansible#7922]
#### Description
While running the uninstall playbook I got this error:
```
failed: [node.example.com] (item=/var/lib/docker) => {"changed": false, "item": "/var/lib/docker", "msg": "rmtree failed: [Errno 16] Device or resource busy: '/var/lib/docker/containers'"}
```
After manually stopping docker on the server: ```systemctl stop docker```
And re-running the playbook, the uninstall process was successful.

##### Version
```
$ ansible --version
ansible 2.5.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/ylifshit/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.14 (default, Mar 14 2018, 13:36:31) [GCC 7.3.1 20180303 (Red Hat 7.3.1-5)]
```
```
$ git describe
openshift-ansible-3.9.14-1
```

##### Steps To Reproduce

1. Install openshift using ansible playbook. 
2. call: ```ansible-playbook -i my-inventory openshift-ansible/playbooks/adhoc/uninstall.yml```

##### Additional Information

* Running ansible from Fedora27 (not inside the cluster)
* Cluster is made of RHEL7.4 servers
* inventory: https://gist.github.com/yuvalif/777b9b992cf11a3505e6577eb664357c

**Comment by bbeaudoin** [comment]
It was noted `/var/lib/docker/` was not being cleaned up in a previous issue (https://github.com/openshift/openshift-ansible/issues/3082).

The playbook was changed to add `/var/lib/docker` to the list of remaining files to remove between release-3.7 and release-3.9 but did not use the same method by which `/var/lib/origin` and `/var/lib/etcd` were removed.

```
git diff origin/release-3.7..origin/release-3.9 uninstall.yml
[...]
+    - /etc/pki/ca-trust/source/anchors/openshift-ca.crt
     - /etc/sysconfig/atomic-openshift-node
     - /etc/sysconfig/atomic-openshift-node-dep
     - /etc/sysconfig/openshift-node-dep
@@ -288,23 +312,38 @@
     - /etc/systemd/system/origin-node-dep.service
     - /etc/systemd/system/origin-node.service
     - /etc/systemd/system/origin-node.service.wants
+    - /var/lib/docker
+
```
Having a `docker-storage-setup` like the following will trigger the current uninstall playbook bug:
```
STORAGE_DRIVER=overlay2
DEVS=/dev/sdb
CONTAINER_ROOT_LV_NAME=dockerlv
CONTAINER_ROOT_LV_SIZE=100%FREE
CONTAINER_ROOT_LV_MOUNT_PATH=/var/lib/docker
VG=dockervg
```

**Comment by bbeaudoin** [comment]
When testing deployments where variables are added to or removed from an inventory, subsequent installations may fail due to host facts not being removed from a previous execution. Suggesting cleaning up cached host facts stored on install.

ansible-playbook all -i <inventory> -m file -a 'path=/etc/ansible/facts.d/openshift.fact state=absent'

**Comment by whitingjr** [comment]
Seeing this in the released OCP 3.10.21. Though that's to be expected when reading [comments](https://github.com/openshift/openshift-ansible/pull/8823#issuecomment-398471716).
 This [workaround](https://github.com/whitingjr/openshift-ansible/tree/issue-%237922) seems to work for me.
